spec/mailers/bulk_upload_mailer_spec.rb

```diff
@@ -46,7 +46,7 @@
           filename: bulk_upload.filename,
           log_type: "lettings",
           upload_timestamp: bulk_upload.created_at.to_fs(:govuk_date_and_time),
-          success_description: "The lettings 2023/24 data you uploaded has been checked. The 0 logs you uploaded are now complete.",
+          success_description: "The lettings #{bulk_upload.year_combo} data you uploaded has been checked. The 0 logs you uploaded are now complete.",
```

Comment:
Do we test anywhere that year_combo returns what we expect?

Reply:
Good spot - have added some.